libs/vm/src/module.cpp

```diff
@@ -242,7 +242,8 @@ Module::Module()
       .CreateSerializeDefaultConstuctor<>()
       //      .CreateConstuctor<>()
       .CreateConstuctor<Ptr<String>>()
-      .CreateMemberFunction("signedTx", &Address::HasSignedTx);
+      .CreateMemberFunction("signedTx", &Address::HasSignedTx)
+      .CreateMemberFunction("display", &Address::AsString);
```

Comment:
This change is not necessary, but useful, I was thinking about to add new standalone  `toString(...)` function for address. but this was way simpler and it aligns with what we have in native code `ledger::Address` type.